plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/starlark/psi/statements/StarlarkStatementContainer.kt

```diff
@@ -0,0 +1,5 @@
+package org.jetbrains.bazel.languages.starlark.psi.statements
+
+interface StarlarkStatementContainer {
+  fun getStatementLists(): List<StarlarkStatementList>
```

Comment:
why are there multiple lists?

Reply:
This is in draft mode and not quite ready for review yet, but thanks for taking a look.

An `if` statement has multiple child `StarlarkStatementList`s, one for each branch.